Example/Example/EditorDemoController.swift

```diff
@@ -11,11 +11,11 @@ class EditorDemoController: UIViewController {
 
     fileprivate var mediaErrorMode = false
 
-    fileprivate(set) lazy var formatBar: Aztec.FormatBar = {
+    fileprivate(set) lazy var formatBar: Aztec.FormatBar = { [unowned self] in
```

Comment:
Mind reverting this one?. Lazy var's closure is @noescape, and thus, does not retain self.

[Ref. Here](https://stackoverflow.com/questions/38141298/lazy-initialisation-and-retain-cycle)